core/flags.rs

```diff
@@ -3,19 +3,15 @@
 use crate::libdeno::deno_set_v8_flags;
 use libc::c_char;
 use libc::c_int;
-use std::ffi::CStr;
 use std::ffi::CString;
-use std::mem;
 use std::vec::Vec;
 
 /// Pass the command line arguments to v8.
 /// Returns a vector of command line arguments that V8 did not understand.
 /// Translates --v8-options into a --help flag for V8.
-pub fn v8_set_flags(args: Vec<String>) -> Vec<String> {
+pub fn v8_set_flags(args: Vec<String>) {
```

Comment:
I don't think you need to change this interface. You can just ignore the return value in CLI ?

Reply:
You're right, reverted